schedule/models/events.py

```diff
@@ -42,11 +42,15 @@ class Event(models.Model):
     title = models.CharField(_("title"), max_length = 255)
     location = models.CharField(_("location"), max_length = 255)
     description = models.TextField(_("description"), null = True, blank = True)
-    creator = models.ForeignKey(User, null = True, verbose_name=_("creator"))
+    creator = models.ForeignKey(User, null = True,
+        verbose_name=_("creator"), on_delete=models.CASCADE)
```

Comment:
do we need to delete the Event if we delete the User of a model.SET_NULL is enough?

Reply:
I didn't think about it at all, models.CASCADE is the default behaviour. The goal of this pull request was to remove the RemovedInDjango20Warning, not to modify models' behaviour.

Maybe we can add a ticket for that.